backend/controllers/cellMove/viewCellSharingAssessmentDetails.js

```diff
@@ -0,0 +1,37 @@
+const moment = require('moment')
+const { serviceUnavailableMessage } = require('../../common-messages')
+const { putLastNameFirst, hasLength } = require('../../utils')
+const { getBackLinkData } = require('./cellMoveUtils')
+
+module.exports = ({ elite2Api, logError }) => async (req, res) => {
+  const { offenderNo } = req.params
+
+  try {
+    const { firstName, lastName, assignedLivingUnit } = await elite2Api.getDetails(res.locals, offenderNo, true)
```

Comment:
I think the unspoken rule is to group more than one call into a Promise.all() if possible.